services/galley/test/integration/API/Util.hs

```diff
@@ -731,12 +732,14 @@ postConvWithRemoteUsers ::
   Maybe ClientId ->
   NewConv ->
   TestM (Response (Maybe LByteString))
-postConvWithRemoteUsers u c n =
+postConvWithRemoteUsers u c n = do
+  let mock =
+        ("get-not-fully-connected-backends" ~> NonConnectedBackends mempty)
+          <|> mockReply ()
```

Comment:
i would make `postConvWithRemoteUsersGeneric` take a `mock`, and implement `postConvWithRemoteUsersGeneric`, `postConfWithRemoteUsersNonFullyConnected` in terms of that.